skbeam/core/mask.py

```diff
@@ -143,19 +143,13 @@ def ring_blur_mask(img, r, alpha, bins, mask=None):
     img: 2darray
         The  image
     r: 2darray
-        The  array which maps pixels to Q space
+        The  array which maps pixels to bins
     alpha: float or tuple or, 1darray
         Then number of acceptable standard deviations, if tuple then we use
         a linear distribution of alphas from alpha[0] to alpha[1], if array
         then we just use that as the distribution of alphas
-    rmax: float
-        The maximum radial distance on the detector
-    pixel_size: float
-        The size of the pixels, in the same units as rmax
-    distance: float
-        The sample to detector distance, in the same units as rmax
-    wavelength: float
-        The wavelength of the x-rays
+    bins: list
+        The bin edges
     mask: 1darray
```

Comment:
The binned_statistic needs it to be 1D I think, do you want me to note it in the docs?